googlecloud-plugins/docs/BigQuery-batchsource.md

```diff
@@ -0,0 +1,72 @@
+# Google BigQuery Batch Source
+
+
+Description
+-----------
+Batch source to use Google Cloud Platforms's [BigQuery](https://cloud.google.com/bigquery/docs/) as a Source.
+
+Use Case
+--------
+This source is used whenever you need to read from a Google BigQuery table or a BigQuery result.
+
+
+Properties
+----------
+**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**projectID:** The ID of the project in Google Cloud. (Macro-enabled)
+
+**jsonKeyFilePath:** The credential JSON key file path. (Macro-enabled)
+See Google's documentation on [Service account credentials](https://cloud.google.com/storage/docs/authentication#generating-a-private-key) for details.
+Note that user should upload his credential file to the same path of all nodes.
+
+**inputTableId:** The BigQuery table to read from, in the form '<projectId (optional)>:<datasetId>.<tableId>'.
```

Comment:
Since you refer to this as `inputTable`, why not just use that as the property, instead of `inputTableId`?